_posts/2019-09-10-twenty-is-the-new-thirty-comparing-current-and-historical-wgs-accuracy-across-coverage.markdown

```diff
@@ -0,0 +1,193 @@
+Deepak Grover’s submission which used GATK plus custom analysis and was awarded
+best Indel accuracy, and the proof-of-concept version DeepVariant submission.
+
+![figure2]({{ site.baseurl }}/assets/images/2019-09-10/figure2.png)
+
+Relative to the GATK Best Practices and Best Practices with VQSR submissions,
+the recent runs of both GATK4 and DeepVariant are much higher in accuracy. Here
+the current DeepVariant is able to match the 50x GATK Best Practices submission
+at a coverage of around 17x.
+
+There are two possible explanations. Accuracies as a whole may have improved
+substantially since 2016 (DeepVariant’s accuracy has continued to improve
+through the open-source release and subsequent versions). Alternatively, many of
+the Best Practices submissions may not have been executed in an ideal manner (we
+note that other pFDA GATK submissions have a similar error number to the one
+chosen in this chart). Because GATK3 requires a license for use by commercial
+entities, it is difficult for us to check these two possibilities ourselves.
+
+Whichever possibility is correct, this observation is motivation to assess the
+accuracy of your pipeline with Genome in a Bottle. If your analysis pipeline has
+not been updated in several years, it is likely worth considering what you could
+achieve in cost or accuracy with newer methods (and we hope that when you do so
+you will consider DeepVariant for your use cases).
+
+## Assessing error types across coverage ranges
+
+Next we investigate how variant callers balance between false positives and
+negatives as coverage drops. The next two charts break down false positives
+(reference sites called as a variant of any genotype), false negatives (variant
+sites of any genotype called as reference), and genotype errors (heterozygous
+variants misclassified as homozygous and vice versa) for SNPs and Indels.
+
+![figure3]({{ site.baseurl }}/assets/images/2019-09-10/figure3.png)
+
+![figure4]({{ site.baseurl }}/assets/images/2019-09-10/figure4.png)
+
+We also plot the relative proportion of each error class for DeepVariantv0.8 and
+GATK4-HC.
+
+![figure5]({{ site.baseurl }}/assets/images/2019-09-10/figure5.png)
+
+![figure6]({{ site.baseurl }}/assets/images/2019-09-10/figure6.png)
+
+As coverages decreases from 30x, both GATK4-HC and DeepVariant v0.8 behave in
+qualitatively similar ways. False negatives increase with lower coverage,
+especially for SNPs at less than 20x coverage.
+
+At high coverages, GATK4-HC and DeepVariant diverge in error profile.
+DeepVariant maintains a similar proportion of false positives and false
+negatives at higher coverages, while for GATK, higher coverage mostly does not
+decrease its false positive rate and instead reduces its false negatives.
+
+## Genomic regions contributing to differences in accuracy
+
+To understand the differences across coverages, we looked into stratifications
+of accuracy across genomic regions from the extended evaluation of hap.py. There
```

Comment:
Augment `hap.py` with a link to https://github.com/Illumina/hap.py

Reply:
Updated both. Thanks!
https://google.github.io/deepvariant/posts/2019-09-10-twenty-is-the-new-thirty-comparing-current-and-historical-wgs-accuracy-across-coverage/